selfdrive/car/ford/carcontroller.py

```diff
@@ -89,8 +166,21 @@ def update(self, CC, CS, now_nanos):
       gas = accel
       if not CC.longActive or gas < CarControllerParams.MIN_GAS:
         gas = CarControllerParams.INACTIVE_GAS
+
       stopping = CC.actuators.longControlState == LongCtrlState.stopping
-      can_sends.append(fordcan.create_acc_msg(self.packer, self.CAN, CC.longActive, gas, accel, stopping, v_ego_kph=40 * CV.MPH_TO_KPH))
+
+      # Calculate targetSpeed
+      targetSpeed = actuators.speed
+      if not CC.longActive and hud_control.setSpeed:
+        targetSpeed = hud_control.setSpeed
+
+      precharge_actuate, brake_actuate = actuators_calc(self, accel)
+      brake = accel
+      if brake < 0 :
+        brake_clip = self.brake_actutator_target - self.brake_actutator_stdDevLow
+        brake = interp(accel, [ CarControllerParams.ACCEL_MIN, self.brake_converge_at, brake_clip], [CarControllerParams.ACCEL_MIN, self.brake_converge_at, self.brake_0_point])
```

Comment:
Instead of sending the brake `accel` at the moment the actuator is enabled - harsh initial brake, we start from `brake_clip` and we'll reach to match the `accel` value when `accel` is `brake_converge_at`